roles/setup_lvm_lv/defaults/main.yml

```diff
@@ -0,0 +1,43 @@
+debug: False
+
+# Metalk8s defaults lv/FS options
+# This will be combine with the possible specific options
+metalk8s_lvm_lv_defaults_fstype: "ext4"
+metalk8s_lvm_lv_defaults_fs_opts: "-m 0"
+metalk8s_lvm_lv_defaults_force: False
+metalk8s_lvm_lv_defaults_mount_opts: "defaults,noatime"
+
+# Path prefix on the host for the mountpoint of the default LVM LVs
+metalk8s_host_path_vg_metalk8s: '/mnt/vg_metalk8s'
+# Check vg metalk8s_lvm_extra_vgs in setup_lvm_vg role
+# if the variable exists
+# metalk8s_storageclass_<vg>: 'my-new-storage-class'
+# metalk8s_host_path_<vg>: '/mnt/myhostpath'
+
+metalk8s_lvm_lvs:
+  lv01:
+    size: 52G
+  lv02:
+    size: 5G
+  lv03:
+    size: 11G
+
+# Add the extra LVM LV that you want to be managed by this role.
+# Use the following format
+# <VG>:
+#   lv01:
+#     size: 50G
+#     fstype: xfs
+#     force: True
+#     fs_opts: '-m 0'
+#     mount_opts: 'defaults,noatime'
+#     labels: {
+#       'scality.com/mylabel': 'mycustomlabel'
+#     }
+#   lv02:
+#     size: 10G
+#
+# Only the "size" attribute is mandatory.
+# The others attribute value will have the values specified in
+# metalk8s_lvm_lv_defaults variable
+metalk8s_lvm_extra_lvs: {}
```

Comment:
Confused about this one. At some point, we define the 'default' VG in which `metalk8s_lvm_lvs` will be created. However, we also support creation of other VGs, right? And LVs in them? Given what's above, I guess this means one can set up something like

```
my_special_vg:
  lv01: {...}
```

So, why not use the same for the 'default' VG?

```
vg_metalk8s:
    lvfoo: {...}
```
and merge the 'default' ones, after asserting there are no naming conflicts between both dicts.